docs/compiler.md

```diff
@@ -24,6 +24,7 @@ a dialect of Go rather than a complete port of the language:
     overhead for all contracts. This can easily be mitigated by first storing values
     in variables and returning the result.
  * lambdas are supported, but closures are not.
+ * maps are supported, but valid map keys are booleans, integers and byte arrays with length <= 64
```

Comment:
strings? For a regular Go developer 'byte array' looks similar to 'byte slice' and that can't be a map key which makes this developer wonder what 'byte arrays' are we talking about here.